applications/serial_lte_modem/doc/MQTT_AT_commands.rst

```diff
@@ -133,7 +133,12 @@ Response syntax
 
 ::
 
-   #XMQTTCON: <client_id>,<url>,<port>[,<sec_tag>]]
+   #XMQTTCON: <status>,<client_id>,<url>,<port>[,<sec_tag>],<keepalive>]
+
+* The ``<status>`` value is an integer.
```

Comment:
```suggestion
* The ``<status>`` value is an integer.
  It can accept one of the following values:
```

Reply:
Just realized that this one is slightly wrong. Should be:
```suggestion
* The ``<status>`` value is an integer.
  It can have one of the following values:
```